features/system-color.yml

```
@@ -0,0 +1,10 @@
+name: system-color
```

This seems to be a bit more "natural" in use than the hyphenated form. Also, see my commentary on the description:

```suggestion
name: System colors
```